File: C++/easy/grains/grains.cpp
#include "grains.h"

namespace grains {
  unsigned long long square(const int& num_square) {
    unsigned long long result{1};
    for (int i{1}; i < num_square; ++i) result *= 2;
    return result;
  }

  unsigned long long total(void) {
    return (square(64) - 1) * 2 + 1;
  }
}  // namespace grains

File: Python/easy/grains/grains.py
def square(number):
    if number < 1 or number > 64:
        raise ValueError("square must be between 1 and 64")
    result = 1
    while number > 1:
        result = result * 2
        number -=  1
    return result


def total():
    result = 0
    i = 1
    while (i <= 64):
        result += square(i)
        i += 1
    return result
